src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2812,7 +2812,7 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Remoting loopback tests.Can execute command in a disconnected session
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 225
225:     Invoke-Command @parameters

```

Reply:
Hm! Interesting. This might be a transient failure, or it might actually be related... But I don't think it's shown up before so I'll assume transient for now & reassess when we do full review.

Reply:
Looks like it was transient after all. 🎉